src/olympia/devhub/forms.py

```diff
@@ -730,14 +730,19 @@ class AdminForm(happyforms.ModelForm):
     _choices = [(k, v) for k, v in amo.ADDON_TYPE.items()
                 if k != amo.ADDON_ANY]
     type = forms.ChoiceField(choices=_choices)
+    reputation = forms.ChoiceField(
+        label=_(u'Reputation'),
+        choices=[('', '')] + [(i, unicode(i)) for i in range(-3, 4)])
```

Comment:
I wonder if the label in the template can be expanded with more explanation?  Being explicit about what a negative value does, etc.
Or, random idea of the day: replace the numbers with something like '-3 Terrible; '-2 Very Bad'; '-1 Bad'; '0 No Reputation'; '1 Good'; '2 Very Good'; '3 Excellent'.